### DISCUSSION TITLE
Will airflow 2.8.0 be supported by the latest version of the helm chart?

### DISCUSSION THREAD

**Discussion by mahmoudghonimfinout** [airflow-helm/charts#824] [Questions & Answers] [answered]
Airflow release --> https://airflow.apache.org/docs/apache-airflow/stable/release_notes.html#airflow-2-8-0-2023-12-18 
The helm supported version currently is 2.7.x --> https://github.com/airflow-helm/charts/tree/main/charts/airflow#airflow-version-support

**Comment by thesuperzapper** [comment, marked as answer]
@mahmoudghonimfinout thanks for raising this, we are discussing this in https://github.com/airflow-helm/charts/issues/814.

Please follow that discussion, but it looks like 2.8 might work fine with the existing 8.8.0 chart version.
Please help us test, if you are able.
